sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -841,6 +841,76 @@ class VersionsSuite extends SparkFunSuite with Logging {
       }
     }
 
+    test(s"$version: Insert into/overwrite external avro table") {
```

Comment:
Is there a reason to have this in `VersionSuite`?

Reply:
Could you add `SPARK-19878` in a test case name?

Reply:
I am fine to keep it in `VersionSuite`, since it is related to Hive.